proto/prysm/v1alpha1/validator-client/keymanager.proto

```diff
@@ -16,33 +14,6 @@ option java_outer_classname = "KeymanagerProto";
 option java_package = "org.ethereum.validator.accounts.v2";
 option php_namespace = "Ethereum\\Validator\\Accounts\\V2";
 
-// RemoteSigner service API.
-//
-// Defines a remote-signing keymanager which manages eth2
-// validator accounts and can sign respective messages.
-service RemoteSigner {
-    // ListPublicKeysResponse managed by a remote signer.
-    rpc ListValidatingPublicKeys(google.protobuf.Empty) returns (ListPublicKeysResponse) {
-        option (google.api.http) = {
-            get: "/accounts/v2/remote/accounts"
-        };
-    }
-
-    // Sign a remote request via gRPC.
-    rpc Sign(SignRequest) returns (SignResponse) {
-        option (google.api.http) = {
-            post: "/accounts/v2/remote/sign"
-        };
-    }
-}
-
-// ListPublicKeysResponse contains public keys
-// for the validator secrets managed by the remote signer.
-message ListPublicKeysResponse {
-    // List of 48 byte, BLS12-381 validating public keys.
-    repeated bytes validating_public_keys = 2;
-}
-
 // SignRequest is a message type used by a keymanager
 // as part of Prysm's accounts v2 implementation.
 message SignRequest {
```

Comment:
It looks like you can also delete `SignRequest` and `SignResponse`

Reply:
it's still used in the code right now even though the remote is not. you can do a search on the type